### PR TITLE
misc: fix spelling

### PR DESCRIPTION
**Pull request by jbampton**

### CODE REVIEW

File: main/offapi/com/sun/star/awt/XListBox.idl
Comment (Pilot-Pirx) [inline main/offapi/com/sun/star/awt/XListBox.idl:1]: This is a historical typo in our API. I think we would need to address both versions and set the wrong one to "deprecated".

@ardovm What do you think?

Comment (Pilot-Pirx) [reply main/offapi/com/sun/star/awt/XListBox.idl:1]: https://www.openoffice.org/api/docs/common/ref/com/sun/star/awt/XListBox.html#isMutipleMode

Comment (ardovm) [reply main/offapi/com/sun/star/awt/XListBox.idl:1]: Being it a long standing API, IMHO we should be very very motivated in order to change it.
I agree that duplicating it is a feasible way to go, for the moment.

**Review by ardovm — CHANGES_REQUESTED** [review]
Please do not substitute API names.

**Review by Pilot-Pirx — APPROVED** [review]
I think we are good now!

File: main/winaccessibility/source/UAccCOM/MAccessible.h
Comment (ardovm) [inline main/winaccessibility/source/UAccCOM/MAccessible.h:1]: I would normally be very reluctant to rename methods as we are doing here.
However, these methods do not seem to be called anywhere, so renaming them should not break anything.
We should rather understand why this code is here, given that it is not used.
But let's make one step at a time.

**Review by ardovm — APPROVED** [review]
Approved. Thanks!

**Pull request merged**